### PR TITLE
fix(medusa): Use get for creating fulfillments

### PR DESCRIPTION
**Pull request by pKorsholm**

### CODE REVIEW

File: packages/medusa/src/api/routes/admin/orders/create-fulfillment.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ export default async (req, res) => {
         relations: ["fulfillments"],
       })
     const existingFulfillmentMap = new Map(
-      existingFulfillments.map((fulfillment) => [fulfillment.id, fulfillment])
+      existingFulfillments.map((fulfillment) => [fulfillment.id, true])
```
Comment (adrien2p) [inline packages/medusa/src/api/routes/admin/orders/create-fulfillment.ts:118]: todo: With those changes, should we use a set instead of a map?

**Review by adrien2p — CHANGES_REQUESTED** [review]

**Review by adrien2p — APPROVED** [review]
LGTM

**Review by srindom — APPROVED** [review]
LGTM!

**Comment by srindom** [comment]
Should we maybe add an integration test?

**Pull request merged**